stdlib/public/Python/PythonLibrary.swift

```diff
@@ -82,9 +84,13 @@ public struct PythonLibrary {
   }
 }
 
-// Methods of `PythonLibrary` required to set a given the Python version.
+// Methods of `PythonLibrary` required to set a given Python version.
 public extension PythonLibrary {
   static func useVersion(_ major: Int, _ minor: Int? = nil) {
+    precondition(!librarySymbolsLoaded, """
+      Error: \(#function) should not be called after any Python library
```

Comment:
In order for this error to appear as a single line, could you please make this line end with a space followed by a backslash?